### DISCUSSION TITLE
Problem with using FileGlobLivestream

### DISCUSSION THREAD

**Discussion by TheRedM** [scivision/PyLivestream#96] [Q&A]
When i tried FileGlobLivestream the stream is just black screen with audio, i streamed at restream.io just to get a little more info about the stream, the resolution is 0x0 with just audio playing

**Comment by scivision** [comment]
This may be an issue with the media file or FFmpeg itself. I would try copy-pasting the command line into Terminal without using PyLivestream

**Comment by scivision** [comment]
Note: with PyLivestream >= 2.0 this script is replaced by:

```sh
python -m pylivestream.glob
```
